app/assets/stylesheets/application.scss

```diff
@@ -242,7 +242,7 @@ footer {
   .container {
     margin-bottom: 36px;
     white-space: nowrap;
-    overflow-x: scroll;
+    overflow-x: auto;
```

Comment:
What's this for?

Reply:
To remove this weird scrollbar showing up in chrome.

![image](https://user-images.githubusercontent.com/16774200/97269429-88b75300-1853-11eb-9303-21cdea93c677.png)

Reply:
Ohh that was a scrollbar! Amazing.